src/main/scala/lambdanet/Model.scala

```diff
@@ -317,6 +318,58 @@ case class Model(
       PredictionResults(prediction, srcLines)
     }
 
+    def predictOnProjectWithGModules(
```

Comment:
Instead of copying the code and making small changes, we should refactor and reuse the original code to avoid duplication; otherwise the code will become very hard to maintain in the future :(

Reply:
Is it ok that I create a function that does a common implementation of these two function and call it from these function

Reply:
Or make gModules: Vector[GModule] = null argument optional and handle it with null check